### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [naman14/Timber#464]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the app [Timber Music](https://play.google.com/store/apps/details?id=naman14.timber&hl=it&gl=US)
3. Activate the registration from the Accessibility Scanner and use the Timber Music app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201118_155420](https://user-images.githubusercontent.com/25429624/99549773-4ef0ec80-29ba-11eb-9cd7-fde6d47e9819.png)
